src/phoenix/db/migrations/env.py

```diff
@@ -14,10 +13,10 @@
 # access to the values within the .ini file in use.
 config = context.config
 
-# Interpret the config file for Python logging.
-# This line sets up loggers basically.
-if config.config_file_name is not None:
-    fileConfig(config.config_file_name, disable_existing_loggers=False)
+# # Interpret the config file for Python logging.
+# # This line sets up loggers basically.
+# if config.config_file_name is not None:
+#     fileConfig(config.config_file_name, disable_existing_loggers=False)
```

Comment:
Is this reading the `alembic.ini` config?

Reply:
Yes it was, didn't know if you wanted them nixed or commented since I saw a lot of comments in the file. WIll remove

Reply:
Done